public/static/js/projects.js

```diff
@@ -171,6 +171,7 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location)
           return v.alias + v.variant + v.task;
         });
         $scope.privateVars = data.ProjectVars.private_vars || {};
+        $scope.githubHookId = data.ProjectVars.github_hook_id || 0;
```

Comment:
Should the id be a string?

Reply:
Nope, it's the numeric hook id that github provides. If it's 0, the project is unconfigured. If not, it's configured